vowpalwabbit/search.cc

```diff
@@ -628,7 +628,7 @@ void reset_search_structure(search_private& priv)
   if (priv.adaptive_beta)
   {
     float x = -log1pf(-priv.alpha) * (float)priv.total_examples_generated;
-    static const float log_of_2 = (float)0.6931471805599453;
+    static constexpr float log_of_2 = (float)0.6931471805599453;
```

Comment:
Aside, and well out of scope of this change: We should do an audit of the magic numbers in VW, and see if we can unify some/any of them.

Reply:
Agreed, now they are easy to find with constexpr :)